.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,6 +7,11 @@
 - [ ] Yes
 - [ ] No
 
+## This should yield a new module release
+
+- [ ] Yes
+- [ ] No
```

Comment:
I believe it's better to not use check-boxes (`[ ]`) here, because the GitHub interface interprets them as "to do" items and shows related statuses. With Yes/No options being alternative checkboxes, it always looks like there's something incomplete.
```suggestion
<!-- Leave one of the options -->

- Yes
- No
```

Reply:
Thanks, it is removed now and I will makes those changes for the template as well :+1: